mitodl/mitxpro
# Added developer readme for sheets feature

#### What are the relevant tickets?
No ticket

#### What's this PR do?
Adds instructions for setting up your development environment for hacking on the sheets feature

#### How should this be manually tested?
First, read the [general README](https://github.com/mitodl/mitxpro/blob/master/sheets/README.md) to familiarize yourself with what this feature does and why it exists.

Then go through the dev setup readme, check for any failures or unclear/inaccurate instructions
